app/models/version.js

```diff
@@ -142,7 +142,13 @@ export default class Version extends Model {
   @computed('loadDocsBuildsTask.lastSuccessful.value')
   get hasDocsRsLink() {
     let docsBuilds = this.loadDocsBuildsTask.lastSuccessful?.value;
-    return docsBuilds && docsBuilds.length !== 0 && docsBuilds[0].build_status === true;
+    return (
+      docsBuilds &&
+      docsBuilds.length !== 0 &&
+      docsBuilds[0] !== null &&
+      docsBuilds[0] !== undefined &&
+      docsBuilds[0].build_status === true
+    );
```

Comment:
```suggestion
    return docsBuilds?.[0]?.build_status === true;
```

we might be able to use optional chaining here to simplify the code a little bit. can you check if this passes the tests too? :)